packages/backend/src/services/test-run.ts

```diff
@@ -86,6 +86,10 @@ const testRun = async (options: TestRunOptions) => {
       nextStepId = nextStep?.id
     }
   }
+
+  await flow.$query().patch({
```

Comment:
wait sorz is it me or this will never occur? should this be done at the start instead